integrations/sveltekit/tsconfig.json

```diff
@@ -0,0 +1,17 @@
+{
+  "extends": ["../../tsconfig.json", "./.svelte-kit/tsconfig.json"],
+  "compilerOptions": {
+    "allowJs": true,
+    "checkJs": true,
+    "esModuleInterop": true,
+    "forceConsistentCasingInFileNames": true,
+    "resolveJsonModule": true,
+    "skipLibCheck": true,
+    "sourceMap": true,
+    "strict": true,
+    "module": "NodeNext",
+    "moduleResolution": "NodeNext",
+    "declaration": true,
+    "declarationMap": true
+  }
+}
```

Comment:
any chance we can remove a few of those compilerOptions, or do we really need to overwrite them all?

Reply:
Removed almost all of them -- I kept `declaration`, `declarationMap`, and `sourceMap` so that click-through for TypeScript works (pet peeve of mine: getting sent to .d.ts files when clicking on a function). If we want to remove those to reduce package size, we can, but that shouldn't practically matter, given that none of it is runtime code and it all gets removed by bundlers. (And this is a primarily server-side library so it matters less in the first place.)